### DISCUSSION TITLE
Opheebop

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#2784] [General]
![Week 2 Opheebop](https://user-images.githubusercontent.com/79092777/147713634-a9d00bf1-8205-4a07-a0d5-741305c4c5b4.gif)
![Week 5 Opheebop](https://user-images.githubusercontent.com/79092777/147713637-7facc72d-9f29-4ded-8e49-0eb48d0edbbb.gif)

**Comment by KiraBytesGit** [comment]
Opheebop
